src/content/teams/13_geospatial.mdx

```diff
@@ -0,0 +1,27 @@
+---
+name: Geospatial Team
+description: Making Nix and NixOS the first choice for users of geospatial software
+members:
+  - name: Ivan Mincik
+    discourse: imincik
+    title: Team lead
+  - name:  Nikolay Korotkiy
+    discourse: sikmir
+    title:
+contact:
+  - name: Discourse
+    href: https://discourse.nixos.org/t/week-in-geospatial-team/37035
```

Comment:
```suggestion
    href: https://discourse.nixos.org/t/creating-nix-geospatial-team/23454
```

Reply:
```suggestion
    href: https://discourse.nixos.org/t/creating-nix-geospatial-team/23454/9
```

That's the post where you make the announcement